bin/stack_functions

```diff
@@ -431,6 +431,14 @@ function reset_system {
     fi
 }
 
+function setup_corp_faucet_config {
+    echo "** setting corp config"
```

Comment:
Keep debugging echo similar style to elsewhere (i.e. nothing else is prefixed by **)

Reply:
Actually I was going to remove but forgot.. Removed now